### PR TITLE
Hide help button from UBE

### PR DESCRIPTION
**Pull request by SiobhyB**
Addresses https://github.com/wordpress-mobile/gutenberg-mobile/issues/4339

## Related PRs and Installable Builds

* `Gutenberg`: https://github.com/WordPress/gutenberg/pull/37221
* `WPiOS`: https://github.com/wordpress-mobile/WordPress-iOS/pull/17644
* `WPAndroid`: https://github.com/wordpress-mobile/WordPress-Android/pull/15674

## Description

As described in https://github.com/wordpress-mobile/gutenberg-mobile/issues/4339, a help button is currently displaying in the bottom right-hand corner of the unsupported block editor (UBE) in both the Android and iOS apps. The button can be seen while logged into WordPress.com. When tapped on, the button isn't working as expected on iOS and prompts users to navigate away from the editor on Android, creating the potential for edits to be lost.

With this PR, the button is hidden from view in order to avoid confusion and potential loss of edits. A high-level overview of the code changes can be found in the `Types of changes` section of [the Gutenberg PR](https://github.com/WordPress/gutenberg/pull/37221).

## Testing

Please refer to [the Gutenberg PR](https://github.com/WordPress/gutenberg/pull/37221) as the "central PR" with the most up-to-date testing instructions. 

<hr />

PR submission checklist:

- [ ] I have considered adding unit tests where possible.
- [ ] I have considered if this change warrants user-facing release notes [more info](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/Release-notes.md) and have added them to [RELEASE-NOTES.txt](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) if necessary.

### CODE REVIEW

File: resources/unsupported-block-editor/external-style-overrides.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* Remove \'help\' button from the editor. */
```
Comment (twstokes) [inline resources/unsupported-block-editor/external-style-overrides.css:1]: Nice! Looks like the right selector in my testing. 👍

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
 		"eslint": "^6.8.0",
 		"eslint-plugin-eslint-comments": "3.1.2",
 		"eslint-plugin-import": "2.18.2",
+		"eslint-plugin-jest": "^25.3.0",
```
Comment (twstokes) [inline package.json]: @SiobhyB is this dependency upgrade required for this feature? Thanks!

**Comment by twstokes** [comment]
Great work @SiobhyB! This is working well. 🙇  So far I've added some comments and questions to all the PRs but WPAndroid. Thank you.

File: resources/unsupported-block-editor/external-style-overrides.css (hunk 1 above)
Comment (SiobhyB) [reply resources/unsupported-block-editor/external-style-overrides.css:1]: Thank you!

File: package.json (hunk 2 above)
Comment (SiobhyB) [reply package.json]: It's not, no, thanks for spotting that! I had some trouble getting tests to pass with this PR, so kept running `npm install` and this was a change that got committed as part of that. I've updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4352/commits/79e5caf1ae84111d2c1157f67b3754c685af04e4.

**Comment by SiobhyB** [comment]
Thank you so much for the review so far @twstokes! 🙇‍♀️ I've gone ahead to respond to all of your comments now, looking forward to hearing any further thoughts.

I also wanted to note that the `Check Correctness` test keeps failing on this PR, which is the reason you'll see a lot of changes to the `package-lock.json` in the commit history. I'll make sure to clean the commit history and resolve any conflicts with `package-lock.json` before merging.

**Review by twstokes — APPROVED** [review]
LGTM @SiobhyB. 👍

**Pull request merged**